EIPS/Network upgrade postmortem template.md

```diff
@@ -0,0 +1,92 @@
+---
+eip: TBD
+title: Recommendation for creating Ethereum network upgrade analysis/postmortem report
+author: Pooja Ranjan 
+discussions-to: 
+status: Draft
+type: Meta
```

Comment:
I think this should be informational.

Reply:
Sorry, I take that back. I think *this* EIP should be meta, but it should specify that EIPs that follow this template are *informational*.